socs/agents/ucsc_radiometer/agent.py

```diff
@@ -0,0 +1,122 @@
+from os import environ
+
+import requests
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import Pacemaker, TimeoutLock
+
+
+class UCSCRadiometerAgent:
+    """Monitor the PWV Flask Server.
```

Comment:
"Flask server" is an implementation detail. Everywhere in these docs can we refer to "radiometer web server" wherever "PWV flask server" is mentioned?